repo/packages/E/etcd/0/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "etcd",
+  "version": "0.0.1",
+  "scm": "https://github.com/mesosphere/etcd-mesos",
+  "maintainer": "support@mesosphere.io",
+  "description": "A distributed consistent key-value store for shared configuration and service discovery.",
+  "framework": true,
+  "images": {
+    "icon-small": "https://downloads.mesosphere.io/etcd-mesos/assets/etcd-small.png",
+    "icon-medium": "https://downloads.mesosphere.io/etcd-mesos/assets/etcd-medium.png",
+    "icon-large": "https://downloads.mesosphere.io/etcd-mesos/assets/etcd-large.png"
+  },
+  "tags": ["mesosphere", "framework", "consensus"],
+  "preInstallNotes": "In order for etcd to start successfully all resources must be available in the cluster including ports, CPU shares and RAM.\nWe recommend a minimum of 3 nodes with 1 CPU share and 128 MB of RAM available for use by the etcd service.\nNote that the service is alpha and there may be bugs, including possible data loss, incomplete features, incorrect documentation or other discrepancies.",
+  "postInstallNotes": "Once the cluster initializes (<1 minute if offers are available), etcd proxies may connect by passing the argument -discovery-srv=etcd.mesos (or -discovery-srv=<framework-name>.mesos if you're not using the default), and you may discover live members by querying SRV records for _etcd-server._tcp.<framework-name>.mesos"
```

Comment:
for postInstallNotes, this is a lot of detail and it would be best IMO to have a link to these notes.   I'm willing to merge into multiverse as is... lets update before GA.